Sources/PackageModel/SwiftSDKs/SwiftSDK.swift

```diff
@@ -417,19 +439,44 @@ public struct SwiftSDK: Equatable {
         )
     }
 
+    /// Creates a Swift SDK with the specified properties.
+    @available(*, deprecated, message: "use `init(hostTriple:targetTriple:toolset:pathsConfiguration:xctestSupport:)` instead")
+    public init(
+        hostTriple: Triple? = nil,
+        targetTriple: Triple? = nil,
+        toolset: Toolset,
+        pathsConfiguration: PathsConfiguration,
+        supportsTesting: Bool
+    ) {
+        let xctestSupport: XCTestSupport
+        if supportsTesting {
+            xctestSupport = .supported
+        } else {
+            xctestSupport = .unsupported(reason: nil)
+        }
+
+        self.init(
+            hostTriple: hostTriple,
+            targetTriple: targetTriple,
+            toolset: toolset,
+            pathsConfiguration: pathsConfiguration,
+            xctestSupport: xctestSupport
+        )
+    }
+
     /// Creates a Swift SDK with the specified properties.
     public init(
```

Comment:
```suggestion
    package init(
```

Reply:
Let's keep the new initializer hidden for now. We need to understand the exact use cases for exposing `SwiftSDK` as `public` in the first place before encouraging further adoption of it, especially as its API is not stable.